runtime-sdk/modules/evm/src/evm_backend.rs

```diff
@@ -2,6 +2,7 @@
 use std::cell::RefCell;
 
 use evm::backend::{Apply, ApplyBackend, Backend as EVMBackend, Basic, Log};
+use oasis_core_runtime::common::crypto::hash::Hash;
```

Comment:
```suggestion
use oasis_runtime_sdk::core::common::crypto::hash::Hash;
```
And move to the import group below.